Node-DC-EIS-client/node_dc_eis_testurls.py

```diff
@@ -320,8 +359,8 @@ def main_entry(url, request_num, url_type, log_dir, phase, interval,
       sys.exit(1)
 
   if url_type == 1:
-    get_url(url, request_num, log, phase, accept_header)
+    get_url(url,url_type,request_num, log, phase, accept_header)
   if url_type == 2:
-    post_url(url,request_num,log,phase)
+    post_url(url,url_type,request_num,log,phase)
   if url_type == 3:
-    delete_url(url,request_num,log,phase)
\ No newline at end of file
+    delete_url(url,url_type,request_num,log,phase)
```

Comment:
the github seems to be complaining about the lack of new line at the end. Should we add one?